### DISCUSSION TITLE
What means "BlockedByResponse" CefErrorCode?

### DISCUSSION THREAD

**Discussion by assopri** [cefsharp/CefSharp#3818] [Q&A]
Hi! I get error when using LoadUrlAsyncResponse. CefErrorCode is -27: BlockedByResponse . Didn`t find the description in file. What this error means?

**Comment by amaitland** [comment]
See https://github.com/cefsharp/CefSharp/blob/1cdb21960c99c32ca8be8f4b81707cd911bf5406/CefSharp/Enums/CefErrorCode.cs#L178

**Reply by assopri** [reply]
Thank you!
